mediacloud/mediawords/dbi/downloads.py

```diff
@@ -0,0 +1,388 @@
+"""Various helper functions for downloads, including storing and fetching content.
+
+This module includes various helper function for dealing with downloads.
+
+Most importantly, this module has the store_content and fetch_content
+functions, which store and fetch content for a download from the pluggable
+content store.
+
+The storage module is configured in mediawords.yml by the
+mediawords.download_storage_locations setting.
+
+The three choices are:
+
+* 'postgresql', which stores the content in a separate postgres table and
+  optionally database
+* 'amazon_s3', which stores the content in amazon_s3
+* 'databaseinline', which stores the content in the downloads table downloads
+  are no longer stored in `databaseinline', only read from.
+
+The default is 'postgresql', and the production system uses Amazon S3.
+
+This module also includes extract and related functions to handle download
+extraction.
+"""
+
+import random
+import re
+import typing
+
+from mediawords.db import DatabaseHandler
+from mediawords.key_value_store import KeyValueStore
+import mediawords.util.extract_text
+import mediawords.util.html
+from mediawords.util.log import create_logger
+
+log = create_logger(__name__)
+
+# PostgreSQL table name for storing raw downloads
+RAW_DOWNLOADS_POSTGRESQL_KVS_TABLE_NAME = 'raw_downloads'
```

Comment:
i personally find it a lot more confusing to have to go to a different file to find some random constant rather than looking for it at the top of the package I'm writing.

Reply:
It's not a global constant, it's a local one, intended to be used only by the module. Possible improvement would be to prefix its name with `__` to communicate that fact.